### ISSUE TITLE
If Task on record is deleted but EP Task remains, updating other task status fails with error

### ISSUE THREAD

**Issue by judisohn** [SalesforceFoundation/Cumulus#2495]
support case: [15733215](https://org62.lightning.force.com/one/one.app#/sObject/5000M00000gVgx4QAC/view) 

Issue replicated. Steps to repeat:

1. Create EP Template with 3 tasks. 2 dependent on 1, 3 dependent on 2 (settings on template do not matter, as long as the tasks have dependencies)
2. Create EP to a Contact (object doesn't matter)
3. On the Contact's record, delete the final task (which should be 'Waiting on Dependent Task')
4. Set status on first 'Not Started' task to 'Completed'
5. Error:

![image](https://cloud.githubusercontent.com/assets/4614343/23045753/ee936004-f473-11e6-94d0-a56240bd91a7.png)

It may not be an uncommon use case to edit individual task records, apart from the template's more generic tasks.

**Comment by judisohn** [comment]
**lurch: add

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x800000050EpmAAE" target="_blank">W-017798</a>

**Comment by hwong103** [comment]
Just wanted to add as part of this bug, adding a new task to an existing template causes the same issue for records that do not have this newly created task in their plans.

**Comment by judisohn** [comment]
Thanks for the update @hwong103

**Comment by mrbelvedere** [comment]
Included in beta release 3.93 (Beta 1)

**Comment by mrbelvedere** [comment]
Included in production release rel/3.93
